### DISCUSSION TITLE
Community plugins missing in NetBeans 15 plugin catalog?

### DISCUSSION THREAD

**Discussion by norrisjeremy** [apache/netbeans#4849] [Q&A]
Does anybody know why most community plugins are missing from the NetBeans 15 plugin catalog (https://netbeans.apache.org/nb/plugins/15/catalog.xml.gz)?

**Comment by neilcsmith-net** [comment]
Delays in verification.  There are discussions underway about improving this situation.  In the meantime, you could add additional plugin centres for 13 or 14.

**Comment by norrisjeremy** [comment]
It's disappointing to hear that 7 weeks after the NetBeans 15 release, this still isn't resolved. And NetBeans 16 is just around the corner already...

**Comment by geertjanw** [comment]
Will you join in to verify plugins?

Gj

**Comment by norrisjeremy** [comment]
What changed in NetBeans 15 that would cause every plugin to require verification?
There's nothing documented in the [release notes](https://github.com/apache/netbeans/releases/tag/15).

**Comment by geertjanw** [comment]
So, you're saying we shouldn't verify plugins at all?

Or only some of them, which ones, and will you verify these?

**Reply by errael** [reply]
> So, you're saying we shouldn't verify plugins at all?

What he said was that he'd like to know what changed that every existing plugin would need to be verified again.

The context of his statement makes that pretty clear. Why do you purposefully misrepresent to make a point.

**Comment by norrisjeremy** [comment]
I'm suggesting that it is difficult to know what to verify when there is a distinct lack of documentation to indicate what changed to potentially cause plugins to no longer operate correctly.

Also if there is a a lack of manpower to help verify plugins, then it would certainly help if it was better publicized.
As I stated earlier, I don't see anything mentioned in the release notes, there's also nothing published on the [blog](https://blogs.apache.org/netbeans/) and nothing mentioned here on the Discussions board besides this particular discussion.

Also you fail to mention how a community member is even supposed to participate in this plugin verification process. Where would I (or anyone else) go to help participate in this process?

But hey, I get it: it's easier to post one-liners and argue with a user instead of supplying constructive answers.

**Comment by neilcsmith-net** [comment]
@norrisjeremy I totally agree with you and share your frustration.  This situation needs resolving before we release NetBeans 16.  The plugin centre for that is totally empty, and we haven't had a fully available plugin catalogue since NetBeans 13.  We are trying to find a better answer before NetBeans 16 is released, but please bear with us!

In the meantime, the workaround is to add the older plugin centres as suggested earlier.
